INSTALLATION.md

```diff
@@ -91,6 +91,27 @@ Follow these steps to get the api running using npm
    ```sh
    npm run start
    ```
+## Automated Installation
+
+This mode of installation helps to run all the installation steps automatically.
+
+You must have the following installed on your system:
+- [NodeJS 14.0.0](https://www.nodejs.org) or higher 
+- [Python 3](https://www.python.org) or higher
+
+To start with the installation process,
+
+1. Clone this repo to your local machine
+
+   ```sh
+   git clone https://github.com/PalisadoesFoundation/talawa-api
+   cd talawa-api
+   ```
+2. Run the following command to start the installation process. 
+
+   ```sh
+   npm run setup
+   ```
 
```

Comment:
Can you please add a small note here and put the reference of `setup/README.md` like `For more information on automated installation visit [here](url)`

Reply:
Done